main.go

```diff
@@ -0,0 +1,97 @@
+package main
+
+import (
+	"bufio"
+	"flag"
+	"fmt"
+	"os"
+	"os/exec"
+)
+
+var (
+	command string
+	resFile string
+)
+
+func main() {
+	if len(os.Args) < 2 {
+		printUsage()
+	}
+
+	flag.Parse()
+
+	outFile, err := os.Create(resFile)
```

Comment:
File open should specify the method and the file attributes. 
In this particular case the file will being written only, therefore it should not be open for read as well.
`Create creates the named file with mode 0666 (before umask), truncating it if it already exists. If successful, methods on the returned File can be used for I/O; the associated file descriptor has mode O_RDWR. If there is an error, it will be of type *PathError.`